balance/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "tower-h2-balance"
+version = "0.1.0"
+authors = ["Oliver Gould <ver@buoyant.io>"]
+
+[dependencies]
+futures = "0.1"
+http = "0.1"
+h2 = "0.1"
+
+tower-h2  = { version = "0.1", path = ".." }
+
+tower-balance = { git = "https://github.com/tower-rs/tower", branch = "ver/balance-track-rsp-h2" }
```

Comment:
I’m a little amazed this works. I would have thought that the same branch needs to be referenced for all tower deps.

Reply:
This crate only depends on `tower-balance` (from the tower repo)